Libplanet.Tests/Blocks/BlockHeaderTest.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Immutable;
+using System.Globalization;
+using Libplanet.Blocks;
+using Xunit;
+
+namespace Libplanet.Tests.Blocks
+{
+    public class BlockHeaderTest : IClassFixture<BlockFixture>
+    {
+        private BlockFixture _fx;
+
+        public BlockHeaderTest(BlockFixture fixture) => _fx = fixture;
+
+        [Fact]
+        public void ValidateTimestamp()
+        {
+            DateTimeOffset now = DateTimeOffset.UtcNow;
+            string future = (now + TimeSpan.FromSeconds(901))
```

Comment:
That can make sense. but I prefer specifying detail directly as a spec in test case instead of reusing constant from target project. (it isn't strong opinion 😅 )